hypothesis-python/src/hypothesis/database.py

```diff
@@ -178,10 +177,7 @@ def save(self, key, value):
         mkdir_p(self._key_path(key))
         path = self._value_path(key, value)
         if not os.path.exists(path):
-            suffix = binascii.hexlify(os.urandom(16))
-            if not isinstance(suffix, str):  # pragma: no branch
-                # On Python 3, binascii.hexlify returns bytes
-                suffix = suffix.decode("ascii")
+            suffix = binascii.hexlify(os.urandom(16)).decode("ascii")
```

Comment:
In Python 3.6, this can get a bit simpler:

```python
import secrets

...
suffix = secrets.token_hex(16)
```

which returns a string natively.

Reply:
Soon!  (ish!)